website/docs/user_guide/rbac.md

```diff
@@ -119,7 +119,51 @@ A user group consists of the following:
 - a **name** (required)
 - a **description** (optional)
 - a list of users (optional)
+- a list of SSO groups to sync from (optional)
 
 Groups do nothing on their own. They must be given a role on a project to assign permissions. You can assign both standard roles and custom project roles to groups.
 
 While a user can only have one role in a given project, a user may belong to multiple groups, and each of those groups may be given a role on a project. In the case where a given user is given permissions to a project through more than one group, the user will inherit most permissive permissions of all their groups in that project.
+
+## User Group SSO Integration
+
+:::info availability
+
+User group syncing is planned to be released in Unleash 4.18 and will be available for enterprise customers.
+
+:::
+
+User groups also support integration with your Single Sign-On (SSO) provider. This allows you to automatically assign users to groups when they log in through SSO. Check out [_how to set up group SSO sync_](../how-to/how-to-set-up-group-sso-sync.md) for more details.
```

Comment:
Wonder if we can clarify this a bit? So that it's not generic "details" (because they should be in the reference)?

```suggestion
User groups also support integration with your Single Sign-On (SSO) provider. This allows you to automatically assign users to groups when they log in through SSO. Check out [_how to set up group SSO sync_](../how-to/how-to-set-up-group-sso-sync.md) for a step-by-step walkthrough.
```

Reply:
Sure, that looks better to me